doc/sphinxman/source/sapt.rst

```diff
@@ -425,6 +467,60 @@ Specific open-shell SAPT0 keywords
 .. include:: autodir_options_c/sapt__sapt_mem_safety.rst
 .. include:: autodir_options_c/sapt__coupled_induction.rst
 
+.. index:: SAPT; SAPT(DFT)
+
+
+SAPT(DFT)
+^^^^^^^^^
+
+In general, SAPT(DFT) should provide more accurate
+interaction components, and overall interaction energies, than SAPT0.
+While being significantly more computationally expensive than SAPT0,
+SAPT(DFT) can still be applied to medium-sized or large systems. The
+SAPT(DFT) program was successful in computations of systems with up
+to 2000 basis functions, and the code should be scalable to 3000 basis
+functions. Like higher-order SAPT, SAPT(DFT) requires sufficient memory
+to hold :math:`2ovN_aux` doubles in core.
```

Comment:
Slight rewording, suggest taking a look at other paragraphs.

In general, SAPT(DFT) should provide more accurate interaction energy components, and overall interaction energies, than SAPT0. The drawback is SAPT(DFT) method is more computationally demanding than SAPT0, SAPT(DFT) can still be applied to medium-sized or large systems. The SAPT(DFT) module was employed successfully in computations of systems with up to 2000 basis functions, and the code should be scalable to 3000 basis functions. Like higher-order SAPT, SAPT(DFT) requires sufficient memory to hold :math:`2ovN_aux` doubles in memory.